### PR TITLE
Add Make build command for integration test

### PR DESCRIPTION
**Pull request by lkolacek**
Support local integration testing with Makefile build command

Signed-off-by: Ladislav Kolacek <lkolacek@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- n/a Code coverage from testing does not decrease and new code is covered
- n/a New code has type annotations
- [x] Docs updated (if applicable)
- n/a Docs links in the code are still valid (if docs were updated)
- n/a [Issue](https://github.com/containerbuildsystem/cachi2/issues/13) that tracks breaking changes to the OSBS2 integration is updated (if applicable)

### CODE REVIEW

**Comment by lkolacek** [comment]
Does it look usable to you? Example: http://pastebin.test.redhat.com/1085356

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,10 @@ venv:
 	venv/bin/pip install tox
 	venv/bin/pip install -e .
 
+test-with-build:
```
Comment (chmeliik) [inline Makefile:18]: Following the `test-unit` naming, maybe this could be `test-integration`? Otherwise this looks great :+1:

Comment (lkolacek) [reply Makefile]: Ack

**Review by taylormadore — COMMENTED** [review]
The change LGTM :+1: 

Do we want to update the docs? I see we have a section for unit-tests

**Comment by lkolacek** [comment]
> The change LGTM +1
> 
> Do we want to update the docs? I see we have a section for unit-tests

Great idea Taylor. Is this description sufficient?

**Review by chmeliik — APPROVED** [review]
LGTM :tada:

**Review by ejegrova — APPROVED** [review]

**Review by taylormadore — APPROVED** [review]
LGTM :tada:

**Pull request merged**